static/templates/recent_topic_row.hbs

```diff
@@ -1,18 +1,43 @@
-<tr id="recent_topic:{{topic_key}}" {{#if unread_count}}class="unread_topic"{{/if}} data-unread-count="{{unread_count}}" data-muted="{{muted}}" data-participated="{{participated}}">
+<tr id="recent_conversation:{{conversation_key}}" {{#if unread_count}}class="unread_topic"{{/if}} data-unread-count="{{unread_count}}" data-muted="{{muted}}" data-participated="{{participated}}">
     <td class="recent_topic_stream">
+        {{#if is_private}}
+        <div class="flex_container flex_container_pm">
```

Comment:
Why do we have this containing `div` with the `flex_container` class only for private messages? I'm wondering if we can shrink the region that has this if/else block, and/or just make the markup more similar, which in turn would make reading the CSS easier.

(I did notice that the topic block does have both stream and PM versions inside a shared `flex_container` div).

Reply:
In `Private message` row we need to show the presence/group indicator for PMs, for this reason I have used the `flex_container` for only PMs. I'll try to shrink it.